config/webpack.server.js

```diff
@@ -70,7 +70,7 @@ module.exports = (env, argv) => {
     module: {
       rules: [
         {
-          test: /\.js$/,
+          test: /\.[tj]sx?$/,
```

Comment:
Is there actually an extension we use that matches this regex? Wouldn't they be ts or tsx here as well?

Reply:
This regex matches .ts, .tsx, .js, and .jsx. But we don't have any js or jsx files, so I suppose it could be simplified

Reply:
This regex matches .ts, .tsx, .js and .jsx, so it should work fine. All the tests are .ts or .tsx though afaik, so I suppose it could be simplified